layers/+spacemacs/spacemacs-navigation/packages.el

```diff
@@ -369,6 +370,48 @@
     :documentation "Smooth scrolling."
     :evil-leader "tv"))
 
+(defun spacemacs-navigation/init-symbol-overlay ()
+  (use-package symbol-overlay
+    :init
+    (progn
+      (setq spacemacs--symbol-overlay-transient-state-doc "
```

Comment:
To align both the transient state string and how it appears when the TS is opened, two caret characters (`^^`) need to be added for every additional key that's missing, to match the entry with the most keys in a column.

For example the current second column:
```
[_N_/_p_]
[_o_]
[_r_]
```
should look like this:
```
[_N_/_p_]
[_o_]^^
[_r_]^^
```

Since `q` is bound to quit, it should be listed as well.
Several other Transient states have two spaces between the columns.

With these changes the transient state looks like this:
```
      (setq spacemacs--symbol-overlay-transient-state-doc "
%s
 [_n_] next    [_N_/_p_] prev       [_d_] def            [_f_/_b_] switch  [_t_] scope
 [_e_] echo    [_o_]^^   unoverlay  [_O_] unoverlay all  [_c_]^^   copy    [_z_] center
 [_s_] search  [_r_]^^   replace    [_R_] rename         ^^^^              [_q_] quit")
```

Before:

![before](https://user-images.githubusercontent.com/13420573/56904301-4a927500-6a9e-11e9-9a7d-5fdc1c2b2efd.png)

After:

![after](https://user-images.githubusercontent.com/13420573/56904319-5120ec80-6a9e-11e9-9891-97393f95e207.png)

Reply:
Hi @duianto . Thanks for the review. I am not that familiar with hydra. I will refine this as you pointed out.